### PR TITLE
Add Organic Maps icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes https://github.com/simple-icons/simple-icons/issues/6094

**Popularity metric:**

Organic Maps now has ~8.7k stars on GitHub which puts it into scope. See https://github.com/organicmaps/organicmaps

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Just resized and optimized it.

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
Couple of issues on this one - would you mind having a look please @CodeShakingSheep?

**Comment by CodeShakingSheep** [comment]
> Couple of issues on this one - would you mind having a look please @CodeShakingSheep?

Ah yes, just updated the source. Any other issues @adamrusted ?

File: icons/organicmaps.svg
Comment (adamrusted) [inline icons/organicmaps.svg:1]: Sorry, yes @CodeShakingSheep - there's a difference between your PR (black) and the source icon (red) below. Looks like it's just been broken in optimization - might be worth another try?
image

Comment (CodeShakingSheep) [reply icons/organicmaps.svg:1]: Strange, I don't know how the difference was created. The source svg only has 1 path. So, the only things I did was removing the <g> element and resizing it. 

Anyway, I just edited the source svg from scratch in Inkscape but when using an optimization precision of 4, I end up with the exact same path. So, I increased the precision to 5 although not sure if that solves the difference. Is it better now?

**Review by adamrusted — CHANGES_REQUESTED** [review]
Still seeing exactly the same difference when using [this image](https://organicmaps.app/logos/green-on-transparent.svg) from the site header as source. Could you try re-generating please @CodeShakingSheep? 🙏
image

**Comment by CodeShakingSheep** [comment]
> Still seeing exactly the same difference when using [this image](https://organicmaps.app/logos/green-on-transparent.svg) from the site header as source. Could you try re-generating please @CodeShakingSheep? 🙏

Thanks for reviewing @adamrusted . I used exactly that image from the website's header. I think there's something odd with the your downloaded svg from this PR. You wrote that black is my PR, right? However, when I zoom into my svg in Inkscape the edge is sharp (not rounded as in your screenshot). See here:
image

Also, when you zoom into the svg on GitHub you can see that it's sharp and not rounded. So, I assume that somehow the svg from this PR was altered while you downloaded it? It's really weird.

**Review by adamrusted — APPROVED** [review]
You were right @CodeShakingSheep - viewing the source file in another software fixed the issue, and it's a perfect match! Thanks for you patience on this one!

**Pull request merged**